include/rocksdb/filter_policy.h

```diff
@@ -41,6 +42,64 @@ class FilterBitsBuilder {
   // The return value of this function would be the filter bits,
   // The ownership of actual data is set to buf
   virtual Slice Finish(std::unique_ptr<const char[]>* buf) = 0;
+
+  // Calculate num of entries fit into a space.
+  virtual int CalculateNumEntry(const uint32_t space) {
+    throw new std::runtime_error("CalculateNumEntry not Implemented");
+  }
+};
+
+class FullFilterBitsBuilder : public FilterBitsBuilder {
```

Comment:
This is external (user facing) header. I'm suggesting to move this to internal (hidden) header.